wownar-react-native/client/package.json

```diff
@@ -9,27 +9,28 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@coinbase/coinbase-sdk": "^4.0.4",
```

Comment:
Would you be able to downgrade to `v0.2.0`? That's the [latest version I see on npm](https://www.npmjs.com/package/@coinbase/coinbase-sdk)